### PR TITLE
mRNA updates

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2, here are some mRNA updates I forgot to add in January. I added a few new references tonight, but some of the text could be curtailed and updated further.

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -596,7 +623,7 @@ The most common adverse reactions reported across all platforms are local site i
...
+For the whole-virus vaccines, CoronaVac has now been approved despite some halts due to reports of adverse reactions [@url://www.nytimes.com/interactive/2020/health/sinovac-covid-19-vaccine.html], and a trial of the SinoPharm WIV04 vaccine in Peru was briefly paused due to safety concerns in relation to neurological symptoms [@url:https://www.nytimes.com/live/2020/12/16/world/covid-19-coronavirus#the-clinical-trials-of-a-chinese-vaccine-resume-in-peru-after-a-volunteers-illness-is-deemed-unrelated], but this was later deemed unrelated to the vaccine, and the trial continued [@clinicaltrials:NCT04612972].
```
Comment (RLordan) [inline content/22.vaccines.md:626]:
```suggestion
For the whole-virus vaccines, CoronaVac has now been approved despite some halts due to reports of adverse reactions [@url:https://www.nytimes.com/interactive/2020/health/sinovac-covid-19-vaccine.html], and a trial of the SinoPharm WIV04 vaccine in Peru was briefly paused due to safety concerns in relation to neurological symptoms [@url:https://www.nytimes.com/live/2020/12/16/world/covid-19-coronavirus#the-clinical-trials-of-a-chinese-vaccine-resume-in-peru-after-a-volunteers-illness-is-deemed-unrelated], but this was later deemed unrelated to the vaccine, and the trial continued [@clinicaltrials:NCT04612972]. 
```

File: content/22.vaccines.md [hunk 2]
```diff
@@ -539,14 +539,13 @@ As of the 2010s, mRNA was still considered a promising technology for future adv
 Therefore, while these technologies elegantly capitalize on decades of research in vaccine development as well as the tools of the genomic revolution, it was largely unknown prior to the SARS-CoV-2 pandemic whether this potential could be realized in a real-world vaccination effort.
 
 **Application to COVID-19:**
-Given the potential for this technology to be quickly adapted for a new pathogen, it has held significant interest for the treatment of COVID-19.
+Given the potential for this technology to be quickly adapted for a new pathogen, it was favoured as a potential vaccine against COVID-19.
```
Comment (rando2) [inline content/22.vaccines.md:542]:
```suggestion
Given the potential for this technology to be quickly adapted for a new pathogen, it was favored as a potential vaccine against COVID-19.
```

File: content/22.vaccines.md [hunk 3]
```diff
@@ -539,14 +539,13 @@ As of the 2010s, mRNA was still considered a promising technology for future adv
...
 Two vaccine candidates in this category emerged with promising phase III results at the end of 2020.
 Both require two doses approximately one month apart.
 The first was Pfizer/BioNTech's BNT162b2, which contains the full prefusion stabilized, membrane-anchored SARS-CoV-2 spike protein in a vaccine formulation based on modified mRNA (modRNA) technology [@doi:10.1101/cshperspect.a014035; @url:https://www.pfizer.com/news/press-release/press-release-detail/pfizer-and-biontech-announce-early-positive-data-ongoing-0].
 <!--To Do: brief history of phase I & phase II (like 1-2 sentences)-->
-mRNA-1273 is comprised by a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
-In the phase I trial, neutralizing activity reached similar levels to that observed in convalescent plasma samples by day 7 after the second dose of RNA-1273 [@doi:10.1056/NEJMoa2022483].
+The second mRNA vaccinec, mRNA-1273 developed by ModernTX, is comprised by a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
```
Comment (rando2) [inline content/22.vaccines.md:548]:
```suggestion
The second mRNA vaccine, mRNA-1273 developed by ModernTX, is comprised by a conventional lipid nanoparticle encapsulated RNA encoding a full-length prefusion stabilized S protein for SARS-CoV-2 [@clinicaltrials:NCT04283461].
```

File: content/22.vaccines.md [hunk 4]
```diff
@@ -557,11 +556,14 @@ See https://greenelab.github.io/covid19-review/ for the most recent version of t
...
+Pfizer/BioNTech's BNT162b2 vaccine and ModernaTX’s mRNA-1273 vaccine, commercially known as Comirnaty and Spikevax, are available in most countries thanks to their rapid development in 2020. 
+In a phase II/III multinational trial, the Pfizer/BioNTech's BNT162b2 vaccine was associated with a 95% efficacy against laboratory-confirmed COVID-19 and with mild-to-moderate local and systemic effects but a low risk of serious adverse effects when the prime-boost doses were administered 21 days apart [@doi:10.1056/NEJMoa2034577].
+The ModernaTX mRNA-1273 vaccine, was the second mRNA vaccine to release phase III results, despite being the first mRNA vaccine to enter phase I clinical trials and publish interim results of their phase III trial a few months later.
```
Comment (rando2) [inline content/22.vaccines.md:561]:
```suggestion
The ModernaTX mRNA-1273 vaccine was the second mRNA vaccine to release phase III results, despite being the first mRNA vaccine to enter phase I clinical trials and publish interim results of their phase III trial a few months later. 
```

File: content/22.vaccines.md [hunk 5]
```diff
@@ -571,6 +573,31 @@ Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, si
 The occurrence of fevers was also lower in individuals who were partially or fully vaccinated, and the duration of symptoms was approximately 6 days shorter.
 <!--Flesh this out, what were the relative rates of strains?-->
 
+Just like the other available COVID-19 vaccines, the efficacy of mRNA vaccines has been challenged by the emergence of variants of concern (VOC). 
+These VOC have gene mutations that code for an altered spike protein and so the antibodies developed resulting from the immunization with the existing vaccines may not be as efficacious, which has caused major concern [@doi:10.1101/2021.11.11.21266068; @url:https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html].
```
Comment (rando2) [inline content/22.vaccines.md:577]:
```suggestion
These VOC have gene mutations that code for an altered spike protein, so the antibodies developed resulting from the immunization with the existing vaccines may not be as efficacious, which has caused major concern [@doi:10.1101/2021.11.11.21266068; @url:https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html].
```

File: content/22.vaccines.md [hunk 6]
```diff
@@ -571,6 +573,31 @@ Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, si
...
+Despite some reports of varying and reduced efficacy of the mRNA vaccines against the Alpha (B.1.1.7), Beta (B.1.351), and Delta (B.1.617.2) variants versus the original SARS-CoV-2 strain or the D614G variant [@DOI:10.1038/s41591-021-01318-5; @doi:10.1038/s41586-021-03398-2; @DOI:10.1001/jama.2021.4388; ], the greatest concern to date has been the Omicron variant (B.1.1.529).which was first identified in November 2021  [@url:https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern; @url:https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html].
```
Comment (rando2) [inline content/22.vaccines.md:578]:
```suggestion
Despite some reports of varying and reduced efficacy of the mRNA vaccines against the Alpha (B.1.1.7), Beta (B.1.351), and Delta (B.1.617.2) variants versus the original SARS-CoV-2 strain or the D614G variant [@DOI:10.1038/s41591-021-01318-5; @doi:10.1038/s41586-021-03398-2; @DOI:10.1001/jama.2021.4388; ], the greatest concern to date has been the Omicron variant (B.1.1.529), which was first identified in November 2021 [@url:https://www.who.int/news/item/26-11-2021-classification-of-omicron-(b.1.1.529)-sars-cov-2-variant-of-concern; @url:https://www.cdc.gov/coronavirus/2019-ncov/science/science-briefs/scientific-brief-omicron-variant.html].
```

Comment (rando2) [reply content/22.vaccines.md:578]: We should see if this URL builds ok, I think the parentheses might cause an issue!

File: content/22.vaccines.md [hunk 7]
```diff
@@ -571,6 +573,31 @@ Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, si
...
+The Omicron variant now accounts for 95% of all infections sequenced in the United States [@url:https://covid.cdc.gov/covid-data-tracker/#variant-proportions] and has been linked to an increased risk of SARS-CoV-2 reinfection [@doi:10.1101/2021.11.11.21266068] and further infection of those who have been vaccinated with the mRNA vaccines [@doi:10.2139/ssrn.3981711].
```
Comment (rando2) [inline content/22.vaccines.md:579]:
```suggestion
As of March 2022, the Omicron variant accounts for 95% of all infections sequenced in the United States [@url:https://covid.cdc.gov/covid-data-tracker] and has been linked to an increased risk of SARS-CoV-2 reinfection [@doi:10.1101/2021.11.11.21266068] and further infection of those who have been vaccinated with the mRNA vaccines [@doi:10.2139/ssrn.3981711]. 
```

File: content/22.vaccines.md [hunk 8]
```diff
@@ -571,6 +573,31 @@ Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, si
...
+Estimates for the mRNA vaccines range from a 2-fold to over a 20-fold drop in neutralisation titres [@doi:10.1101/2021.12.13.21267748], hence the push for third doses of mRNA vaccines in many Western countries. 
+A third mRNA vaccine dose does increase antibody titers, but these levels also wane with time [@doi:10.15585/mmwr.mm7107e2]. 
+Notably, immunocompromised individuals such as cancer patients seem to elecit a sufficient protective immune response against the Omicron variant when they have been boosted with a third dose of either mRNA vaccine, albeit a blunted response [@doi:10.1016/j.ccell.2021.12.014]. 
+While antibody titres do correlate with protection [@doi:10.1126/science.abm3425; @doi:10.1016/j.vaccine.2021.12.006; @doi:10.1016/j.vaccine.2021.05.063; @doi:10.1101/2021.05.21.21257572; @doi:10.1038/s41423-020-00588-2], they are not the only mechanisms of immune protection, for example T cell and non-neutralizing antibody responses may be unaffected or less affected by the new VOC and they warrant further investigation.
```
Comment (rando2) [inline content/22.vaccines.md:590]:
```suggestion
While antibody titres do correlate with protection [@doi:10.1126/science.abm3425; @doi:10.1016/j.vaccine.2021.12.006; @doi:10.1016/j.vaccine.2021.05.063; @doi:10.1101/2021.05.21.21257572; @doi:10.1038/s41423-020-00588-2], they are not the only mechanisms of immune protection; for example, T cell and non-neutralizing antibody responses may be unaffected or less affected by the new VOC and they warrant further investigation.
```

File: content/22.vaccines.md [hunk 9]
```diff
@@ -571,6 +573,31 @@ Overall efficacy of mRNA vaccines was estimated at 91% with full vaccination, si
...
+
+In countries such as Israel, a fourth dose of mRNA vaccines have been introduced in response to the Omicron variant and an intial study in healthcare workers show that the additonal immunization is safe and immunogenic with antibody titres restored to peak-third dose titers.
+No severe illness was reported in the cohort studied (274 versus 426 age-matched controls) and vaccine efficacy against infection was reported at 30% for BNT162b2 and 11% for mRNA-1273 [@doi:10.1101/2022.02.15.22270948].
```
Comment (rando2) [inline content/22.vaccines.md:593]:
```suggestion
No severe illness was reported in the cohort studied (274 versus 426 age-matched controls), and vaccine efficacy against infection was reported at 30% for BNT162b2 and 11% for mRNA-1273 [@doi:10.1101/2022.02.15.22270948].
```

**Review by rando2 — APPROVED** [review]
This is great text @RLordan! I think some of it might end up being added to other manuscripts? But I'll merge it in now and move it as I finalize the text!

File: content/22.vaccines.md (hunk 6 above)
Comment (agitter) [reply content/22.vaccines.md:578]: I edited your suggestion to add curly braces around the URL, which should handle the parenthesis inside correctly.

**Pull request merged**